QuasselDroid/src/main/java/com/iskrembilen/quasseldroid/Network.java

```diff
@@ -289,8 +293,10 @@ public int getCountUsers() {
     }
 
     private void updateTopic(){
-        if(statusBuffer!=null){
-            statusBuffer.setTopic("");
+        if(statusBuffer != null){
+            statusBuffer.setTopic(networkName + " (" + server + ") | "
+                    + Quasseldroid.context.getResources().getString(R.string.users) + ": "
+                    + userList.size() + " | " + Helper.formatLatency(latency, Quasseldroid.context.getResources()));
```

Comment:
This creates double latency formatting, as the latency is also added by the updateSubtitle() method in MainActivity